src/main/app-launcher.ts

```diff
@@ -4,67 +4,65 @@ import { backToMainShortcut, cmd } from "../core/utils.js"
 
 let findAppsAndPrefs = async () => {
   log(`findAppsAndPrefs`)
-  let apps = await fileSearch("", {
-    onlyin: "/",
-    kMDItemContentType: "com.apple.application-bundle",
-  })
-  let manualAppDir = await readdir("/Applications")
-  apps = apps.concat(
-    manualAppDir
-      .filter(app => app.endsWith(".app"))
-      .map(app => `/Applications/${app}`)
-  )
-  // apps = _.uniq(apps.filter(a => !a.includes("Users")))
-  let prefs = await fileSearch("", {
-    onlyin: "/",
-    kind: "preferences",
-  })
-  return {
-    apps,
-    prefs,
+  if (process.platform === "darwin") {
+    let apps = await fileSearch("", {
+      onlyin: "/",
+      kMDItemContentType: "com.apple.application-bundle",
+    })
+    let manualAppDir = await readdir("/Applications")
+    apps = apps.concat(
+      manualAppDir
+        .filter(app => app.endsWith(".app"))
+        .map(app => `/Applications/${app}`)
+    )
+    // apps = _.uniq(apps.filter(a => !a.includes("Users")))
+    let prefs = await fileSearch("", {
+      onlyin: "/",
+      kind: "preferences",
+    })
+    return {
+      apps,
+      prefs,
+    }
+  } else if (process.platform === "win32") {
+    let globalApps = await fileSearch("", {
+      onlyin:
+        '"%ProgramData%\\Microsoft\\Windows\\Start Menu\\Programs"',
+      kind: "application",
+    })
+    let apps = await fileSearch("", {
+      onlyin:
+        '"%AppData%\\Microsoft\\Windows\\Start Menu\\Programs"',
+      kind: "application",
+    })
+    return {
+      apps: [...globalApps, ...apps],
+      prefs: [],
+    }
   }
 }
 let createChoices = async () => {
-  let { fileIconToFile } = await npm("file-icon")
+  let { extractIcon } = await npm("get-app-icon")
```

Comment:
I had to enter .kenv manually and install that dependency. Running from the script was giving me the error: 

```
Unhandled Promise Rejection
Error: File not found: 
    at new WindowsPtyAgent (C:/Users/fernando/AppData/Local/Programs/kit/resources/app.asar/node_modules/node-pty/src/windowsPtyAgent.ts:105:49)
    at new WindowsTerminal (C:/Users/fernando/AppData/Local/Programs/kit/resources/app.asar/node_modules/node-pty/src/windowsTerminal.ts:51:19)
    at Object.spawn (C:/Users/fernando/AppData/Local/Programs/kit/resources/app.asar/node_modules/node-pty/src/index.ts:30:10)
    at C:/Users/fernando/AppData/Local/Programs/kit/resources/app.asar/main.prod.js:2:283089
    at l (C:/Users/fernando/AppData/Local/Programs/kit/resources/app.asar/main.prod.js:2:274939)
    at Generator.<anonymous> (C:/Users/fernando/AppData/Local/Programs/kit/resources/app.asar/main.prod.js:2:276277)
    at Generator.next (C:/Users/fernando/AppData/Local/Programs/kit/resources/app.asar/main.prod.js:2:275302)
    at Zl (C:/Users/fernando/AppData/Local/Programs/kit/resources/app.asar/main.prod.js:2:281962)
    at a (C:/Users/fernando/AppData/Local/Programs/kit/resources/app.asar/main.prod.js:2:284641)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```